### ISSUE TITLE
WordOps broke while updating to v3.9.9.3.

### ISSUE THREAD

**Issue by mardoqueo** [WordOps/WordOps#179]
Installation stops and after that WordOps stopped working. Running the install commands cause the same effect.

```
root@MARDOQUEO-ubuntu-s-3vcpu-1gb-nyc3-01:~# wo update 
Downloading update script        [Done]
updating WordOps, please wait...

Welcome to WordOps install/update script v3.9.9.3

Backing-up WO install
Installing wo dependencies
root@MARDOQUEO-ubuntu-s-3vcpu-1gb-nyc3-01:~#
```

System Information

- [ ] `lsb_release -a`
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco
```
- [ ] `wo info`

```
Traceback (most recent call last):
  File "/usr/local/bin/wo", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3191, in <module>
    @_call_aside
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3175, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3204, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'wo==3.9.8.11' distribution was not found and is required by the application
```

- [ ] `nginx -V`
```
nginx version: nginx/1.16.1 (WordOps Nginx-wo)
built with OpenSSL 1.1.1d  10 Sep 2019
TLS SNI support enabled
configure arguments: --with-cc-opt='-g -O2 -fdebug-prefix-map=/build/nginx-yZZjue/nginx-1.16.1=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -Wdate-time -D_FORTIFY_SOURCE=2' --with-ld-opt='-Wl,-Bsymbolic-functions -Wl,-z,relro -fPIC' --prefix=/usr/share/nginx --conf-path=/etc/nginx/nginx.conf --http-log-path=/var/log/nginx/access.log --error-log-path=/var/log/nginx/error.log --lock-path=/var/lock/nginx.lock --pid-path=/run/nginx.pid --http-client-body-temp-path=/var/lib/nginx/body --http-fastcgi-temp-path=/var/lib/nginx/fastcgi --http-proxy-temp-path=/var/lib/nginx/proxy --http-scgi-temp-path=/var/lib/nginx/scgi --http-uwsgi-temp-path=/var/lib/nginx/uwsgi --build='WordOps Nginx-wo' --with-debug --with-pcre-jit --with-file-aio --with-threads --with-http_ssl_module --with-http_v2_module --with-http_stub_status_module --with-http_realip_module --with-http_v2_hpack_enc --with-http_auth_request_module --with-openssl-opt='enable-ec_nistp_64_gcc_128 enable-tls1_3 no-ssl3-method' --with-http_addition_module --with-http_geoip_module --with-http_gzip_static_module --with-http_mp4_module --with-http_sub_module --without-http_memcached_module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/headers-more-nginx-module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/nginx-auth-pam --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/nginx-cache-purge --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/nginx-echo --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/nginx-upstream-fair --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/ngx_http_substitutions_filter_module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/memc-nginx-module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/srcache-nginx-module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/HttpRedisModule --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/redis2-nginx-module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/ngx_devel_kit --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/set-misc-nginx-module --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/nginx-module-vts --add-module=/build/nginx-yZZjue/nginx-1.16.1/debian/modules/ngx_brotli
```

- [ ] `wo -v`
- [ ] `wp cli info --allow-root`
- [ ] `curl -sL git.io/fjAp3 | sudo -E bash -`

**Comment by mardoqueo** [comment]
`sudo rm /usr/local/bin/wo && wget -qO wo wops.cc && sudo bash wo` not fixing it.

**Comment by mardoqueo** [comment]
Fresh install in fresh server produces same result.

**Comment by VirtuBox** [comment]
Hello @mardoqueo,
thanks for your report. I'm going to investigate on this and to publish a fix asap

**Comment by mardoqueo** [comment]
> Hello @mardoqueo,
> thanks for your report. I'm going to investigate on this and to publish a fix asap

Thank YOU virtu, really appreciate all your effort.

**Comment by VirtuBox** [comment]
Hello @mardoqueo,
issue should be fixed with the last release (v3.9.9.4). Can you confirm it's working on your side ?
Anyway, install process will be updated in the next release and WordOps will be installed with `pip` to avoid similar issues.

**Comment by mardoqueo** [comment]
Hi @VirtuBox I will try some time this week since I'm very busy.
Will get back to you as soon as I test it.

THANKS!!!
